Sources/SwiftPlot/PlotStyleHelpers.swift

```diff
@@ -52,18 +52,34 @@ struct Text : Annotation {
     public var color = Color.black
     public var size: Float = 15
     public var location = Point(0.0, 0.0)
+    public var drawBoundingBox: Bool = false
+    public var bboxBorderSize: Float = 5
+    public var bboxColor = Color.white
     public func draw(renderer: Renderer){
+        if drawBoundingBox {
+            var bboxSize = renderer.getTextLayoutSize(text: text, textSize: size)
+            bboxSize.width += 2 * bboxBorderSize
+            bboxSize.height += 2 * bboxBorderSize
+            let bboxRect = Rect(origin: Point(location.x - bboxBorderSize, location.y - bboxBorderSize),
+                            size: bboxSize)
+            renderer.drawSolidRect(bboxRect,
+                                   fillColor: bboxColor,
+                                   hatchPattern: BarGraphSeriesOptions.Hatching.none)
```

Comment:
Here you can have just `.none`, Swift knows it must be a case of the `BarGraphSeriesOptions.Hatching` enum so you can drop everything up to the `.` of the enumeration case.

Reply:
It also makes it easier to read:

```swift
(... hatchPattern: BarGraphSeriesOptions.Hatching.none)
```
vs
```swift
(... hatchPattern: .none)
```